projects/angular/src/wizard/_wizard.clarity.scss

```diff
@@ -25,129 +27,52 @@
       height: initial;
       overflow: hidden;
       display: flex;
       align-items: flex-start;
       flex-direction: column;
+      gap: wizard-variables.$clr-wizard-default-space;
     }
 
     .modal-header--accessible {
       flex: 0 0 auto;
       width: 100%;
-      // right padding nudged a bit to better align icons
-      @include mixins.css-var(
-        padding,
-        clr-wizard-header-padding,
-        wizard-variables.$clr-wizard-default-space
-          (wizard-variables.$clr-wizard-default-space - variables.$clr_baselineRem_5px) variables.$clr_baselineRem_0_25
-          wizard-variables.$clr-wizard-default-space,
-        variables.$clr-use-custom-properties
-      );
+      padding: 0;
     }
 
     .modal-title {
-      @include mixins.css-var(
-        color,
-        clr-wizard-title-text,
-        wizard-variables.$clr-wizard-title-text,
-        variables.$clr-use-custom-properties
-      );
+      color: wizard-variables.$clr-wizard-title-text;
       display: flex;
       flex-direction: row;
       width: 100%;
-      // align focus style across modal title
-      @include mixins.css-var(
-        font-size,
-        clr-h4-font-size,
-        modal-variables.$clr-modal-title-font-size,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        line-height,
-        clr-h4-line-height,
-        variables.$clr_baselineRem_1_13,
-        variables.$clr-use-custom-properties
-      );
-      margin-top: -1 * variables.$clr_baselineRem_0_13;
     }
 
     .modal-body {
       flex: 1 1 auto;
-      @include mixins.css-var(
-        color,
-        clr-wizard-main-text-color,
-        wizard-variables.$clr-wizard-main-text-color,
-        variables.$clr-use-custom-properties
-      );
+      color: wizard-variables.$clr-wizard-main-text-color;
       width: 100%;
     }
 
-    .modal-footer {
+    .modal-footer.clr-wizard-footer {
       display: block;
       width: 100%;
       flex: 0 0 wizard-variables.$clr-wizard-footer-height;
-      @if variables.$clr-use-custom-properties == true {
-        flex: 0 0 var(--clr-wizard-footer-height);
-      }
-      @include mixins.css-var(
-        height,
-        clr-wizard-footer-height,
-        wizard-variables.$clr-wizard-footer-height,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        min-height,
-        clr-wizard-footer-height,
-        wizard-variables.$clr-wizard-footer-height,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        max-height,
-        clr-wizard-footer-height,
-        wizard-variables.$clr-wizard-footer-height,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        padding,
-        clr-wizard-footer-padding,
-        wizard-variables.$clr-wizard-default-space wizard-variables.$clr-wizard-default-space 0 0,
-        variables.$clr-use-custom-properties
-      );
```

Comment:
I see it was `  --clr-wizard-footer-padding: 0 var(--clr-wizard-main-padding) var(--clr-wizard-main-padding) var(--clr-wizard-main-padding);` in the shim, is this reworked in some way to be the same?